### ISSUE TITLE
Consistent Look and Feel

### ISSUE THREAD

**Issue by sidhshar** [S3Infosoft/s3-loyalty-mobile#7]
Work on making the application look the same Blue and not green in some sections

**Comment by arushi1998** [comment]
For this, we need to make an xml file, inside the drawable folder. We will then set this xml file, in the background attribute of all xml files present in the layout folder, except of the intro page of the loy app.
We will use radial gradient for the background.

**Comment by Varun-L** [comment]
yes i have added the xml files @arushi1998  , can you specify some attractive colors , at present i have put yellow color , suggest some colors for Log In Page , Register Page and Settings page

**Comment by dev-sourabh** [comment]
I corrected the color to blue only in every section
